gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -511,7 +524,18 @@ public void showPolygonInfoDialog() {
     AlertDialog alertDialog = builder.create();
     getStartedBtn.setOnClickListener(
         v -> {
-          alertDialog.dismiss();
+          viewModel
+              .getActiveProject()
+              .ifPresentOrElse(
+                  project -> {
+                    alertDialog.dismiss();
+                    mapContainerViewModel.setSelectedProject(Optional.of(project));
+                    mapContainerViewModel.setSelectedLayer(Optional.of(layer));
+                    mapContainerViewModel.setViewMode(Mode.ADD_POLYGON);
```

Comment:
Nit: ViewMode should be "DRAW_POLYGON", "add" sounds like just the final step.

Reply:
updated